node/resolvers/catalog/index.ts

```diff
@@ -255,16 +361,36 @@ export const queries = {
         ) as any
       }
 
-      response.category = found && found.id as any
+      response.category = found && (found.id as any)
     }
 
     return response
   },
 }
 
+interface Brand {
+  id: string
```

Comment:
```suggestion
  id: ID
```

Reply:
This is a TypeScript interface, not a GraphQL type.